TurkishFilter/sections/specific.txt

```diff
@@ -93,6 +93,7 @@ tekpartizlefullhd.org##a[href][rel="nofollow"]:not([class]) > img
 ||filmkolikhd.net/wp-content/uploads/*/pageskinnnnn.jpg
 /970x250-$image,domain=kriptokoin.com
 /Lunabit.gif$domain=kriptokoin.com
+filmizlehdfull.net###filmoncereklam
```

Comment:
Please use css rules, else user has to wait the timeout until real video appears.

French filter hides skip button, fix:
filmizlehdfull.net#@#.reklamgec